### PR TITLE
Force file to download rather than open in the browser

### PR DESCRIPTION
**Pull request by samuelhwilliams**
### Change description
Force presigned URLs for find download files to actually download, as otherwise the browser may choose to open them in the browser (eg in web excel).

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: core/controllers/async_download.py [hunk 1]
```diff
@@ -147,7 +147,11 @@ def get_presigned_url(filename: str):
 
     url = _S3_CLIENT.generate_presigned_url(
         "get_object",
-        Params={"Bucket": Config.AWS_S3_BUCKET_FIND_DATA_FILES, "Key": filename},
+        Params={
+            "Bucket": Config.AWS_S3_BUCKET_FIND_DATA_FILES,
+            "Key": filename,
+            "ResponseContentDisposition": f'attachment; filename="{filename}"',
+        },
         ExpiresIn=1600,
```
Comment (israr-ulhaq) [inline core/controllers/async_download.py:155]: I think now we can make it to expire in like 60 seconds as you said?

Comment (samuelhwilliams) [reply core/controllers/async_download.py:155]: We can do that in a separate PR - might as well keep this small and focused :) But yes - we can do that. Do you want to make a PR for it?

Comment (israr-ulhaq) [reply core/controllers/async_download.py:155]: Sure, I'll raise one PR for it.

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Pull request merged**